content/financial-aid.html

```diff
@@ -3,6 +3,8 @@
 title: Financial Aid
 ---
 
+<iframe src="https://docs.google.com/forms/d/1JWdkK4pP-USm3duQ-mXdQ6rYEp05lrmqMQ0sSCA5JXw/viewform?embedded=true" width="760" height="800" frameborder="0" marginheight="0" marginwidth="0">Loading...</iframe>
+
```

Comment:
That's what I thought you'd suggested at https://github.com/PyconUK/pyconuk-2016-internaldocs/issues/60#issuecomment-214671406.  Happy to change to a link though, and even happier for you to do it!